impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/Record.java

```diff
@@ -14,11 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating;
+package org.jboss.cdi.tck.tests.full.context.passivating;
+
+import jakarta.enterprise.context.Dependent;
 
 import java.io.Serializable;
 
 @SuppressWarnings("serial")
+@Dependent
```

Comment:
This shouldn't be needed because there is `RecordProducer` class which provides a bean of this type.
However, you need to put `@Dependent` on the `RecordProducer` class instead.